vnext/Microsoft.ReactNative/ReactInstanceSettingsBuilder.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pch.h"
+#include "ReactInstanceSettingsBuilder.h"
+#include "ReactInstanceSettingsBuilder.g.cpp"
+#include "ReactInstanceSettings.h"
+
+namespace winrt::ReactNative {
+using namespace winrt::Microsoft::ReactNative;
+}
+
+namespace winrt::UI {
+using namespace winrt::Microsoft::UI;
+}
+
+namespace winrt::Microsoft::ReactNative::implementation {
+ReactInstanceSettingsBuilder::ReactInstanceSettingsBuilder() {
+  m_reactInstanceSettings = winrt::make<winrt::ReactNative::implementation::ReactInstanceSettings>();
+
+#if _DEBUG
```

Comment:
These DEBUG vs/ not DEBUG settings depends on the app project's config (which will eventually be pulling in release-only M.RN.dll from a nuget). So these initial settings *should* be in the app project, as the library will already be pre-built and this is a developer decision (which they can and do often change).

Reply:
@jonthysell - are you suggesting that developer can choose to enable/disable these settings irrespective of the DEBUG flag enabled or disabled in the app projects config?

Reply:
Yes thats correct.  You need to be able to set these to say debug the JS code on a release build.

Reply:
Yes, as a convenience, the template defaults to enabling the JS debugger in native debug builds, and disabling the JS debugger in native release builds. But devs often change this behavior.